compiler/daml-lf-tools/src/DA/Daml/LF/TypeChecker/Error.hs

```diff
@@ -385,6 +386,8 @@ instance Pretty Error where
       "tried to perform key lookup or fetch on template " <> pretty tpl
     EExpectedOptionalType typ -> do
       "expected list type, but found: " <> pretty typ
+    EExpectedViewType typ -> do
+      "expected monomorphic record type in view type, but found:" <> pretty typ  
```

Comment:
I'm not sure "monomorphic" is exactly the thing we want, after all, `Foo Bool` is monomorphic for this `Foo`:

```haskell
data Foo a = Foo
  { bar : Text
  , baz : a
  }
```

I would say something like this instead, 

```suggestion
      "expected serializable record type without type arguments in view type, but found:" <> pretty typ  
```

I'm not super happy about "without type arguments", but "of kind *" feels too technical.

Reply:
Yes I just noticed that one, I think it's fine to be consistent then.
I also noticed some of these (existing) errors are slightly different between the scala and haskell typecheckers, but that's more of a gardening thing.